supplychain/SupplyChainReferenceOntology.rdf

```diff
@@ -12,63 +13,68 @@
 	<!ENTITY xsd "http://www.w3.org/2001/XMLSchema#">
 ]>
 <rdf:RDF xml:base="https://spec.industrialontologies.org/ontology/supplychain/SupplyChainReferenceOntology/"
+	xmlns="https://spec.industrialontologies.org/ontology/supplychain/SupplyChainReferenceOntology/"
 	xmlns:bfo="http://purl.obolibrary.org/obo/"
+	xmlns:dc="http://purl.org/dc/elements/1.1/"
 	xmlns:dcterms="http://purl.org/dc/terms/"
-	xmlns:iof-av="https://spec.industrialontologies.org/ontology/core/meta/iof-av/"
+	xmlns:iof-av="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"
 	xmlns:iof-core="https://spec.industrialontologies.org/ontology/core/Core/"
 	xmlns:iof-scro="https://spec.industrialontologies.org/ontology/supplychain/SupplyChainReferenceOntology/"
 	xmlns:owl="http://www.w3.org/2002/07/owl#"
 	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
 	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
 	xmlns:skos="http://www.w3.org/2004/02/skos/core#"
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
-
+	
 	<owl:Ontology rdf:about="https://spec.industrialontologies.org/ontology/supplychain/SupplyChainReferenceOntology/">
-		<rdfs:label xml:lang="en-US">Supply Chain Reference Ontology</rdfs:label>
-		<dcterms:abstract xml:lang="en-US">Supply Chain Reference Ontology (SCRO) aims to represent the generic constructs (including classes and properties) related to the domain of supply chain and logistics. SCRO uses BFO as the top-level ontology and IOF Core as the mid-level ontology. The purpose of the ontology is to serve as a foundation for ensuring consistency and interoperability across various supply chain and logistics ontologies that use IOF reference ontolgies.</dcterms:abstract>
-		<dcterms:contributor xml:lang="en-US">Boonserm Kulvatanyou, NIST</dcterms:contributor>
-		<dcterms:contributor xml:lang="en-US">Chris Will, National Center for Ontological Research (NCOR)</dcterms:contributor>
-		<dcterms:contributor xml:lang="en-US">Dusan Sormaz, Ohio University</dcterms:contributor>
-		<dcterms:contributor xml:lang="en-US">Evan Wallace, NIST</dcterms:contributor>
+		<rdfs:label xml:lang="en-us">Supply Chain Ontology</rdfs:label>
+		<dcterms:abstract xml:lang="en-us">The IOF Supply Chain Ontology aims to represent the generic constructs (including classes and properties) related to the domain of supply chain and logistics. The Supply Chain Ontology uses BFO as the top-level ontology and IOF Core as the mid-level ontology. The purpose of the ontology is to serve as a foundation for ensuring consistency and interoperability across various supply chain and logistics ontologies that use IOF reference ontolgies.</dcterms:abstract>
+		<dcterms:contributor xml:lang="en-us">Boonserm Kulvatanyou, NIST</dcterms:contributor>
```

Comment:
I think that https://oagiscore.atlassian.net/wiki/spaces/IOF/pages/4532142081/IOF+Annotation+Property+Guide+V2.2#Summary-of-Annotation-Requirements requires 'en-US'.

This issue should have been found by the hygiene tests, but it was not.
https://github.com/iofoundry/ontology/pull/47 should fix this bug in hygiene tests.